src/fides/api/util/tcf_util.py

```diff
@@ -27,6 +30,16 @@
     TCFPurposeRecord,
     TCFVendorRecord,
 )
+from fides.config.helpers import load_file
+
+_gvl: Optional[Dict] = None
+
+GVL_PATH = join(
+    dirname(__file__),
+    "../../data",
+    "gvl.json",
```

Comment:
I remember there being some trickery when importing static files like this once we get to fidesplus—any chance that will be a problem here?

Reply:
that's a good callout, we'll need to make sure this works!  I tried to avoid hardcoded absolute paths similar to https://github.com/ethyca/fides/pull/3486

Reply:
oh this is a good point, we need to see whether this is packaged up with the python project and if it'd resolve from this relative path. without looking closer, it feels like it may not be, considering that we have to explicitly add the static saas config files to `fidesplus`?

we should be able to give this a test by pushing up an alpha tag to this branch

Reply:
confirmed this works with an alpha tag 🎉